apps/gateway/src/http.ts

```diff
@@ -31,5 +31,10 @@ export const internalRedirect = async (
   }
 
   // @ts-ignore
-  Readable.fromWeb(response.body).pipe(res);
+  const readable = Readable.fromWeb(response.body);
```

Comment:
Can `response.body` ever be null here? If so, we should have a guard.

Reply:
The target endpoint is the download endpoint for auto-drive that always has a body